internal/command/ping/ping.go

```diff
@@ -275,7 +276,6 @@ func run(ctx context.Context) error {
 	for i := 0; count == 0 || i <= count; i++ {
 		select {
 		case <-stp:
-			cancel()
```

Comment:
What is this change for?

Reply:
Good point. Sorry I didn't include the errors reported by govet in the description.

This is it:
```
$ golangci-lint run
internal/command/ping/ping.go:142:2: lostcancel: the cancel function is not used on all paths (possible context leak) (govet)
        ctx, cancel := context.WithCancel(ctx)
        ^
internal/command/ping/ping.go:176:3: lostcancel: this return statement may be reached without using the cancel var defined on line 142 (govet)
                return err
                ^
```